install.sh

```diff
@@ -26,8 +31,8 @@ if [[ $detail_choice == 'y' || $detail_choice == 'Y' ]]; then
 	read -rp "Enter Git email: " email
 
 	# Validate that the email is not empty
-	if [[ -z "$email" ]]; then
-		echo "Error: Git email cannot be empty"
+	if [[ -z "$email" || ! $email =~ $regex ]]; then
+		echo "Error: Your email is either invalid or empty"
 		exit 1
 	fi
```

Comment:
don't you think having different test cases for each error experienced with the email separately, so it should first test if email is empty, display an error and break, if it passes that, it can then verify the email format

Reply:
Personally, it makes no difference, but if you want it in that format, then I'll reformat it that way.

Reply:
I think the error messages should be specific

Reply:
I've changed the messages to be on different based on the specific errors